instruments/instruments/abstract_instruments/comm/loopback_communicator.py

```diff
@@ -21,76 +21,83 @@
 #
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
-##
-##
+#
+#
 
-## IMPORTS #####################################################################
+# IMPORTS #####################################################################
 
 import io
 from instruments.abstract_instruments.comm import AbstractCommunicator
 import sys
 
-## CLASSES #####################################################################
+# CLASSES #####################################################################
+
 
 class LoopbackCommunicator(io.IOBase, AbstractCommunicator):
+
     """
     Used for testing various controllers
     """
-    
+
     def __init__(self, stdin=None, stdout=None):
         AbstractCommunicator.__init__(self)
         self._terminator = '\n'
         self._stdout = stdout
         self._stdin = stdin
-    
-    ## PROPERTIES ##
```

Comment:
Nope, they were just there to look pretty when looking at the source, but pylint likes to complain about them.

I might disable that rule in a .pylintrc file because I kinda like the `##`  for visual distinction